src/domains/dom_runner.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef DOMAIN_CONF_H_
+#define DOMAIN_C0NF_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if !defined(CONFIG_NATIVE_APPLICATION)
```

Comment:
Why do you need this condition here?

Reply:
We don't need domd_cfg in config_native at all. So this ifdef drops extern definition even if native application configuration incdudes this header.

Reply:
After second though - I agree with you. Fixed. Please ignore previous comment